usr/share/rear/build/default/503_store_tty_root_password.sh

```diff
@@ -0,0 +1,3 @@
+# Store TTY_ROOT_PASSWORD for terminal logins
+
+[[ -n "$TTY_ROOT_PASSWORD" ]] && echo "$TTY_ROOT_PASSWORD" > "$ROOTFS_DIR/.TTY_ROOT_PASSWORD"
```

Comment:
Argh!
I see now that also the SSH_ROOT_PASSWORD value
would appear in the log file when the code is run with `set -x`.
So we need to do for SSH_ROOT_PASSWORD and TTY_ROOT_PASSWORD
the same as I did for BACKUP_PROG_CRYPT_KEY by running commands
that would expose those *_ROOT_PASSWORD values
with stderr redirected to /dev/null like
```
{ COMMAND ; } 2>/dev/null
```
Using `2>/dev/null` outisinde of a group command `{ ... ; }` is mandatory
also for a single command inside the group command, cf. the comment
for the `UserInput` function in lib/_input-output-functions.sh that reads (excerpt):
```
The redirection must be done via a compound group command
{ confidential_command ; } 2>/dev/null
even for a single confidential command to ensure
STDERR is redirected to /dev/null also for 'set -x'
otherwise the confidential command and its arguments
would be shown in the log file, for example
{ openssl des3 -salt -k secret_passphrase ; } 2>/dev/null
where the secret passphrase must not appear in the log,
cf. https://github.com/rear/rear/issues/2155
```

Reply:
Well spotted. Addressed via 8bc2b1cccccbd4806feb0bfa5fdcc41d0ca47942.